client/signup/difm/translation-hooks.tsx

```diff
@@ -6,7 +6,6 @@ import {
 	CONTACT_PAGE,
 	ABOUT_PAGE,
 	PHOTO_GALLERY_PAGE,
-	SERVICE_SHOWCASE_PAGE,
 	VIDEO_GALLERY_PAGE,
 	PODCAST_PAGE,
```

Comment:
I haven't removed this for now, since we will be using the Podcast page later. However, Showcase A Service needs to be completely removed.